src/article/editor/RemoveFootnoteCommand.js

```diff
@@ -0,0 +1,15 @@
+import RemoveItemCommand from './RemoveItemCommand'
+
+export default class RemoveCollectionItemCommand extends RemoveItemCommand {
```

Comment:
The file is called RemoveFootnoteCommand. Would like to rename the file or rename the class?

Reply:
I'd say, go for RemoveFootnoteCommand.
When we introduce a similar one we gonna extract a shared impl.